src/lib/api/StoreAPI.ts

```diff
@@ -1,11 +1,12 @@
 import { config } from '@/lib';
-import type {
-  GetAllMerchCollectionsResponse,
-  GetMerchOrdersResponse,
-  GetOneMerchOrderResponse,
-  PublicMerchCollection,
-  PublicOrder,
-  PublicOrderWithItems,
+import {
+  type GetAllMerchCollectionsResponse,
+  type GetMerchOrdersResponse,
+  type GetOneMerchCollectionResponse,
+  type GetOneMerchOrderResponse,
+  type PublicMerchCollection,
+  type PublicOrder,
+  type PublicOrderWithItems,
 } from '@/lib/types/apiResponses';
```

Comment:
You can move all of the `type`s out of the braces right after `import` I think

```suggestion
import type {
  GetAllMerchCollectionsResponse,
  GetMerchOrdersResponse,
  GetOneMerchCollectionResponse,
  GetOneMerchOrderResponse,
  PublicMerchCollection,
  PublicOrder,
  PublicOrderWithItems,
} from '@/lib/types/apiResponses';
```